src/andAndDelete.test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import {
+  tasks, addTasks, createNewTask, repopulateList,
+} from './backEnd';
+import { taskList } from './index1';
+
+describe('It adds and deletes items from the list', () => {
+  const task1 = {
+    description: 'Car wash',
+    complemented: false,
+    index: 0,
+  };
+  const task2 = {
+    description: 'Laundary',
+    complemented: false,
+    index: 1,
+  };
+
+  const taskDescription = 'Pain the house';
+
+  const ul = document.createElement('ul');
+
+  test('it creates a list corresponding inputs', () => {
+    createNewTask(taskDescription);
+
+    expect(tasks[tasks.length - 1].description).toBe(taskDescription);
+
+    ul.appendChild(taskList(tasks[tasks.length - 1]));
+
+    expect(ul.innerHTML.includes(taskDescription)).toBe(true);
+  });
```

Comment:
- [ ] Let's add some negative test case scenarios as well for example what happens if there is no value in input field